oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java

```diff
@@ -559,4 +573,36 @@ private void checkTopNAggregation(Model model, BanyanDBClient client) throws Ban
             }
         }
     }
+
+    @Getter
+    @Setter
+    private static class InstallInfoBanyanDB extends InstallInfo {
+        private DownSampling downSampling;
+        private String tableName;
+        private MetadataRegistry.Kind kind;
+        private String group;
+        private boolean tableExist;
+        private boolean groupExist;
+
+        protected InstallInfoBanyanDB(Model model) {
+            super(model);
+        }
+
+        @Override
+        public String buildInstallInfoMsg() {
+            return "InstallInfoBanyanDB{" +
+                "modelName=" + getModelName() +
+                ", modelType=" + getModelType() +
+                ", timeSeries=" + isTimeSeries() +
+                ", superDataset=" + isSuperDataset() +
+                ", downSampling=" + downSampling.name() +
+                ", tableName=" + tableName +
+                ", kind=" + kind.name() +
+                ", group=" + group +
+                ", allResourcesExist=" + isAllExist() +
+                " [groupExist=" + groupExist +
+                ", tableExist=" + tableExist +
+                "]}";
+        }
```

Comment:
Why not just use `toString` method?

Reply:
I want to avoid the subclass forgetting to override the `toString` since it has a default implementation.